src/server/scripts/World/npcs_special.cpp

```diff
@@ -990,7 +990,7 @@ class npc_doctor : public CreatureScript
                 Reset();
         }
 
-        void PatientSaved(Creature* /*soldier*/, Player* player, Location* point)
+        void PatientSaved(Creature* saved_patient, Player* player, Location* point)
```

Comment:
```suggestion
        void PatientSaved(Creature* savedPatient, Player* player, Location* point)
```

Reply:
Sorry, my Python is showing. I'll fix it.